code/modules/reagents/reagent_containers/food/snacks/grown.dm

```diff
@@ -19,6 +21,12 @@
 		src.pixel_x = rand(-5, 5) * PIXEL_MULTIPLIER
 		src.pixel_y = rand(-5, 5) * PIXEL_MULTIPLIER
 
+/proc/get_special_fruits(var/filter=HYDRO_PREHISTORIC|HYDRO_VOX)
+	. = list()
+	for(var/T in existing_typesof(/obj/item/weapon/reagent_containers/food/snacks/grown))
+		if(initial(T:hydroflags) & filter)
```

Comment:
This uses the : operator which is very haram and explicitly against contribution standards, please find a way to do this without the : operator.